com.unity.render-pipelines.high-definition/Documentation~/Default-Settings-Window.md

```diff
@@ -47,5 +54,5 @@ Use this section to select which custom [Diffusion Profiles](Diffusion-Profile.m
 | --------------------------| ------------------------------------------------------------ |
 | Shader Variant Log Level              | Use the drop-down to select what information HDRP logs about Shader variants when you build your Unity Project. • Disabled: HDRP doesn’t log any Shader variant information.• Only HDRP Shaders: Only log Shader variant information for HDRP Shaders.• All Shaders: Log Shader variant information for every Shader type. |
 | Lens Attenuation Mode                 | Set the attenuation mode of the lens that is used to compute exposure. With imperfect lens some energy is lost when converting from EV100 to the exposure multiplier, while a perfect lens has no attenuation and no energy is lost. |
-| Light Layer Names                     | The name displayed on Lights and Meshes when using [Light Layers](Light-Layers.md). |
-| Decal Layer Names                     | The name displayed on decals and Meshes when using [Decal Layers](Decal.md). |
+| Use DLSS Custom Project Id            | When set to true, a custom project ID can be specified for the NVIDIA Deep Learning Super Sampling module. This setting is only available when the NVIDIA native package is enabled on the project. |
```

Comment:
```Controls whether to use a custom project ID for the NVIDIA Deep Learning Super Sampling module. If you enable this property, you can use **DLSS Custom Project Id** to specify a custom project ID.<br/>This property only appears if you enable the NVIDIA package (com.unity.modules.nvidia) in your Unity project.```

What about when its disabled? Does it just use some default value or generate a unique one for your project?

Reply:
No, so if its disabled, it will utilize the default project ID which is internal in unity.